scale/job/job_type_serializers.py

```diff
@@ -99,55 +54,36 @@ class JobTypeStatusCountsSerializer(serializers.Serializer):
     category = serializers.CharField()
 
 
-class JobTypeDetailsSerializerV5(JobTypeSerializerV5):
-    """Converts job type model fields to REST output for legacy job types."""
-    from error.serializers import ErrorSerializerV5
-
-    interface = serializers.JSONField(default=dict, source='manifest')
-
-    configuration = serializers.JSONField(default=dict)
-    custom_resources = serializers.JSONField(source='convert_custom_resources')
-    error_mapping = serializers.JSONField(default=dict)
-    errors = ErrorSerializerV5(many=True)
-
-    job_counts_6h = JobTypeStatusCountsSerializer(many=True)
-    job_counts_12h = JobTypeStatusCountsSerializer(many=True)
-    job_counts_24h = JobTypeStatusCountsSerializer(many=True)
-
 class JobTypeDetailsSerializerV6(JobTypeSerializerV6):
     """Converts job type model fields to REST output."""
-
     manifest = serializers.JSONField(default=dict)
-
     configuration = serializers.JSONField(source='get_v6_configuration_json')
 
 
 class JobTypeStatusSerializer(serializers.Serializer):
     """Converts job type status model and extra statistic fields to REST output."""
-    job_type = JobTypeBaseSerializerV5()
+    job_type = JobTypeBaseSerializerV6()
     job_counts = JobTypeStatusCountsSerializer(many=True)
 
 
 class JobTypePendingStatusSerializer(serializers.Serializer):
     """Converts job type pending status model and extra statistic fields to REST output."""
-    job_type = JobTypeBaseSerializerV5()
+    job_type = JobTypeBaseSerializerV6()
     count = serializers.IntegerField()
     longest_pending = serializers.DateTimeField()
 
 
 class JobTypeRunningStatusSerializer(serializers.Serializer):
     """Converts job type running status model and extra statistic fields to REST output."""
-    job_type = JobTypeBaseSerializerV5()
+    job_type = JobTypeBaseSerializerV6()
     count = serializers.IntegerField()
     longest_running = serializers.DateTimeField()
 
 
 class JobTypeFailedStatusSerializer(serializers.Serializer):
     """Converts job type failed status model and extra statistic fields to REST output."""
-    from error.serializers import ErrorSerializerV5
 
-    job_type = JobTypeBaseSerializerV5()
-    error = ErrorSerializerV5()
+    job_type = JobTypeBaseSerializerV6()
     count = serializers.IntegerField()
```

Comment:
why remove the error field?

Reply:
Only removed it due the the V5 label 😆  I'm hoping merging with the updates you made to add the missing V6 endpoints will update this properly.